src/XTStyle.cpp

```diff
@@ -38,7 +38,11 @@ void XTStyle::initialize()
     }
     if (!fd)
     {
+#ifdef USING_CARDINAL_NOT_RACK
+        setGlobalStyle(rack::settings::darkMode ? DARK : LIGHT);
```

Comment:
So cardinal users can’t get the mid skin? It’s our most popular one! I wonder if you want to use it instead of light?

Reply:
Ok!